src/asmcnc/apps/drywall_cutter_app/screen_config_filechooser.py

```diff
@@ -0,0 +1,524 @@
+                    show_hidden: False
+                    filters: ['*.json']
+                    on_selection: root.refresh_filechooser()
+                    sort_func: root.sort_by_date_reverse
+                    FileChooserIconLayout
+                        id: icon_layout_fc
+                    FileChooserListLayout
+                        id: list_layout_fc
+
+                ScrollView:
+                    size_hint: 1, 1
+                    pos_hint: {'center_x': .5, 'center_y': .5}
+                    do_scroll_x: True
+                    do_scroll_y: True
+                    scroll_type: ['bars', 'content']
+
+                    Label:
+                        id: metadata_preview
+                        size_hint_y: None
+                        height: self.texture_size[1]
+                        text_size: self.width, None
+                        padding: 10, 10
+                        markup: True
+
+
+        BoxLayout:
+            size_hint_y: None
+            height: 100
+
+            ToggleButton:
+                id: toggle_view_button
+                size_hint_x: 1
+                on_press: root.switch_view()
+                background_color: hex('#FFFFFF00')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    Image:
+                        id: image_view
+                        source: "./asmcnc/skavaUI/img/file_select_list_icon.png"
+                        center_x: self.parent.center_x
+                        y: self.parent.y
+                        size: self.parent.width, self.parent.height
+                        allow_stretch: True 
+
+            Button:
+                id: sort_button
+                size_hint_x: 1
+                on_press: root.switch_sort()
+                background_color: hex('#FFFFFF00')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    Image:
+                        id: image_sort
+                        source: "./asmcnc/skavaUI/img/file_select_sort_down_date.png"
+                        center_x: self.parent.center_x
+                        y: self.parent.y
+                        size: self.parent.width, self.parent.height
+                        allow_stretch: True
+
+            Button:
+                disabled: True
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_release: 
+                    self.background_color = hex('#FFFFFF00')
+                on_press:
+                    self.background_color = hex('#FFFFFFFF')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    
+            Button:
```

Comment:
The code here looks mostly obsolete - can largely be deleted/tidied up if this is just a placeholder/spacer button